tests/run-pass/backtrace-api.rs

```diff
@@ -0,0 +1,24 @@
+// normalize-stderr-test ".*rustlib" -> "RUSTLIB"
```

Comment:
please also normalize away the line and col numbers, these are going to change with almost every rustup otherwise